_events/2021/2021-12-AGM.md

```diff
@@ -26,12 +26,15 @@ Not a member yet? [Join](https://us-rse.org/join/) now!
 
 1. **State of the US-RSE Association: The 2021 year in review and plans for 2022**  
   Ian Cosden, current chair of the Steering Committee, will give an organizational overview of the US-RSE including a review of past events and initiatives, updates on current initiatives, and an outline of some of the future plans and goals for the Association.
-1. **Financial Status and Budget Review**  
-  Chris Hill, US-RSE Treasurer, will provide an update of the US-RSE finances and present a preliminary budget for 2022.
 1. **Updates from Working Groups**  
   A representative from each working group will give a short update on recent activities and plans for 2022.
+  * Diversity, Equity, and Inclusion (DEI)
+  * Education and Training
+  * Website Redesign
```

Comment:
It reads as if it's a list of working groups, is it just the "Website" one rather than also "Redesign"?

Reply:
Good point. I don't know why I wrote redesign, must have been thinking update!

Reply:
Changed to just "Website"